src/Arcus.Security.Providers.AzureKeyVault/KeyVaultSecretProvider.cs

```diff
@@ -28,7 +28,6 @@ public class KeyVaultSecretProvider : ISecretProvider
         /// <summary>
         /// Gets the name of the dependency that can be used to track the Azure Key Vault resource in Application Insights.
         /// </summary>
-        [Obsolete("Will be removed in v2.0")]
         protected const string DependencyName = "Azure key vault";
```

Comment:
```suggestion
        protected const string DependencyName = "Azure Key Vault";
```
?

Reply:
woops :)

Reply:
Ah, no, this is not something we can change or control. I think this is a 'special' name that Azure Application Insights uses to correctly assign the right Azure resource.

Reply:
Yes, this affect the Azure Application Insights dependency call. If we don't use this name, it will show up differently. Not recognized as a Azure Key Vault dependency. This was how it was when we developed dependency tracking.

Reply:
This is for example a set of constants of an AppInsights package, to give you an idea: https://github.com/microsoft/ApplicationInsights-dotnet/blob/main/WEB/Src/DependencyCollector/DependencyCollector/Implementation/RemoteDependencyConstants.cs

Reply:
Thanks for these insights!